src/resolver/resolve_path.zig

```diff
@@ -2126,4 +2126,15 @@ pub fn posixToPlatformInPlace(comptime T: type, path_buffer: []T) void {
     }
 }
 
+/// Like `std.path.isAbsolute`, but allows posix paths on Windows.
+pub fn isAbsoluteCrossPlatform(
```

Comment:
delete this function. it's name is misleading, comment is incorrect, and behavior is already covered in std.fs.path.isAbsolute